docker-compose.yml

```diff
@@ -29,6 +29,28 @@ services:
       - SECRET_POSTGRES_USER
       - SECRET_POSTGRES_PASS
 
+  # These two containers are for running scrape_courses on a schedule in Compute Engine
+  scrape-courses:
+    build:
+      context: .
+      dockerfile: docker/scrape-courses/Dockerfile
+    depends_on:
+      - cloud-sql-proxy
+    volumes:
+      - .:/code
+    container_name: scrape-courses
+    secrets:
+      - SECRET_DJANGO_KEY
+      - SECRET_POSTGRES_USER
+      - SECRET_POSTGRES_PASS
+
+  cloud-sql-proxy:
+    image: gcr.io/cloudsql-docker/gce-proxy:1.11
+    # The 0.0.0.0 is key to having this work on Compute Engine
+    command: /cloud_sql_proxy -instances=$GCP_DB_NAME=tcp:0.0.0.0:3306
+    ports:
+      - 3306:3006
+

```

Comment:
Pretty minor but you should put this container before `scrape-courses` since it depends on `cloud-sql-proxy`